$NetBSD: patch-ab,v 1.3 2012/03/01 05:01:15 obache Exp $

* Not using Solaris builtin make in pkgsrc.
* Honor CFLAGS/CXXFLAGS and avoid to override the optimizer levels.

--- configure.orig	2002-08-20 06:01:54.000000000 +0000
+++ configure
@@ -6095,9 +6095,6 @@ if $ac_make --version 2>/dev/null | grep
         echo "$ac_t""yes" 1>&6
 else
         echo "$ac_t""no" 1>&6
-        if test "$host_vendor" = "sun" ; then
-                { echo "configure: error: "SUN make does not work for building TinySVM. Please install GNU make"" 1>&2; exit 1; }
-        fi
 fi
 
 
@@ -6105,8 +6102,8 @@ fi
 
 
 if test -n "$GCC"; then
-   CFLAGS="-Wall -O9 -funroll-all-loops -finline -ffast-math"
-   CXXFLAGS="-Wall -O9 -funroll-all-loops -finline -ffast-math"
+   CFLAGS="$CFLAGS -Wall -funroll-all-loops -finline -ffast-math"
+   CXXFLAGS="$CXXFLAGS -Wall -funroll-all-loops -finline -ffast-math"
 fi
 
 # On Intel systems with gcc, we may need to compile with -mieee-fp to
